### PR TITLE
GLB 485 Onboarding: Create Editorial section

### PR DESCRIPTION
**Pull request by g-jramirez**
### Context
The onboarding blog requires an Editorial section, containing a selection of posts with marked as "editorial" via a field from Contentful. In order to achieve this, Editorial Grid component needed to be created.

### Description

The following pages where created/updated:
- **Index**: Added another Blog Section with isEditorial flag

The following components are covered:

- **Editorial grid**: A custom grid showing a main post along with reel of 3 posts at most

Preview: [GLB 485](https://deploy-preview-4--keen-goldwasser-abcbdc.netlify.app/)

### Screenshots
imagen

### CODE REVIEW

File: components/EditorialGrid/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const EditorialGrid: React.FC<IEditorialGridProps> = ({ posts }) => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.gridMainPost}>
+        <Post size="large" {...posts[0]} />
+      </div>
+      <div className={styles.gridReel}>
+        {posts.slice(1).map((post, index) => (
+          <Post key={index} size="compact" {...post} />
+        ))}
```
Comment (g-azambrano) [inline components/EditorialGrid/index.tsx:17]: This is mostly fine, but, how can we prevent the layout from breaking? Say, we provide a posts array of 8 elements.

Maybe you need to pick only the next 3 elements that come as part of the array. Thoughts? 🤔

File: components/Post/index.tsx [hunk 2]
```diff
@@ -7,27 +7,29 @@ interface IContentfulPostBody {
   value: string;
 }
 export interface IPostProps {
+  size?: "normal" | "large" | "compact";
```
Comment (g-azambrano) [inline components/Post/index.tsx:10]: This sounds great!

File: components/BlogSection/index.tsx [hunk 3]
```diff
@@ -1,32 +1,43 @@
...
   posts: IPostProps[];
   categories?: string[];
+  isEditorial?: boolean;
 }
 
 const BlogSection: React.FC<IAppProps> = ({
   sectionTitle,
   posts,
   categories,
+  isEditorial,
```
Comment (g-azambrano) [inline components/BlogSection/index.tsx:18]: This is okay for now, but let's discuss at the end of the exercise to discuss some pros vs cons of the `BlogSection` component.

**Review by g-azambrano — APPROVED** [review]
Left minor comments. If you'd like to enter any further improvements, feel free to merge this and enter a new PR.

File: components/EditorialGrid/index.tsx (hunk 1 above)
Comment (g-jramirez) [reply components/EditorialGrid/index.tsx:17]: I'm controlling the amount of posts directly in Blog Section:
imagen

But yes, you are right, this should be ensured inside this component. I'll tackle this in another PR as you suggested.

Comment (g-azambrano) [reply components/EditorialGrid/index.tsx:17]: Oh, great!

**Pull request merged**